src/Engine/ProtoCore/Core.cs

```diff
@@ -932,23 +930,20 @@ public enum ParseMode
 
     public class Core
     {
-        #region RUNTIME_PROPERTIES
 
         /// <summary>
-        ///  These are the list of symbols updated by the VM after an execution cycle
+        /// Properties in under COMPILER_GENERATED_TO_RUNTIME_DATA, are generated at compile time, and passed to RuntimeData/Exe
+        /// Only Core can initialize these
         /// </summary>
-        public HashSet<SymbolNode> UpdatedSymbols { get; private set; }
+        #region COMPILER_GENERATED_TO_RUNTIME_DATA
```

Comment:
Compile directives shouldn't have white space in front of them so they're easier to see